src/activities/discussions/DiscussionTopicEntity.js

```diff
@@ -142,21 +123,67 @@ export class DiscussionTopicEntity extends Entity {
 		];
 		return { action, fields };
 	}
+	/**
+	 * @returns {bool} Sync draft status with topic's parent forum against the draft endpoint
+	 */
+	canSyncDraftWithForum() {
+		return this._entity && this._entity.hasActionByName(Actions.discussions.topic.syncDraftWithForum);
+	}
+	/**
```

Comment:
Not like a big deal but can you add some spacing in between the methods please 👾